chirp/drivers/uvk5.py

```diff
@@ -0,0 +1,1339 @@
+
+            # vox level
+            if element.get_name() == "vox_level":
+                _mem.vox_level = int(element.value)-1
+
+            # mic gain
+            if element.get_name() == "mic_gain":
+                _mem.mic_gain = int(element.value)
+
+            # Channel display mode
+            if element.get_name() == "channel_display_mode":
+                _mem.channel_display_mode = CHANNELDISP_LIST.index(
+                    str(element.value))
+
+            # Crossband receiving/transmitting
+            if element.get_name() == "crossband":
+                _mem.crossband = CROSSBAND_LIST.index(str(element.value))
+
+            # Battery Save
+            if element.get_name() == "battery_save":
+                _mem.battery_save = BATSAVE_LIST.index(str(element.value))
+            # Dual Watch
+            if element.get_name() == "dualwatch":
+                _mem.dual_watch = DUALWATCH_LIST.index(str(element.value))
+
+            # Tail tone elimination
+            if element.get_name() == "tail_note_elimination":
+                _mem.tail_note_elimination = element.value and 1 or 0
+
+            # VFO Open
+            if element.get_name() == "vfo_open":
+                _mem.vfo_open = element.value and 1 or 0
+
+            # Beep control
+            if element.get_name() == "beep_control":
+                _mem.beep_control = element.value and 1 or 0
+
+            # Scan resume mode
+            if element.get_name() == "scan_resume_mode":
+                _mem.scan_resume_mode = SCANRESUME_LIST.index(
+                    str(element.value))
+
+            # Auto keypad lock
+            if element.get_name() == "auto_keypad_lock":
+                _mem.auto_keypad_lock = element.value and 1 or 0
+
+            # Power on display mode
+            if element.get_name() == "welcome_mode":
+                _mem.power_on_dispmode = WELCOME_LIST.index(str(element.value))
+
+            # Keypad Tone
+            if element.get_name() == "keypad_tone":
+                _mem.keypad_tone = KEYPADTONE_LIST.index(str(element.value))
+
+            # Language
+            if element.get_name() == "language":
+                _mem.language = LANGUAGE_LIST.index(str(element.value))
+
+            # Alarm mode
+            if element.get_name() == "alarm_mode":
+                _mem.alarm_mode = ALARMMODE_LIST.index(str(element.value))
+
+            # Reminding of end of talk
+            if element.get_name() == "reminding_of_end_talk":
+                _mem.reminding_of_end_talk = REMENDOFTALK_LIST.index(
+                    str(element.value))
+
+            # Repeater tail tone elimination
+            if element.get_name() == "repeater_tail_elimination":
+                _mem.repeater_tail_elimination = RTE_LIST.index(
+                    str(element.value))
+
+            # Logo string 1
+            if element.get_name() == "logo1":
+                b = str(element.value).rstrip("\x20\xff\x00")+"\x00"*12
```

Comment:
Are these bytes of a logo bitmap? If so, I'm not sure this is useful to expose to the user. Also if these have non-unicode characters then treating them like strings here is going to break as soon as you encounter some combination that triggers a `UnicodeDecodeError`.